src/server/util/RenderEjsHandler.ts

```diff
@@ -0,0 +1,33 @@
+import path from 'path';
+import { renderFile } from 'ejs';
+import type { HttpResponse } from '../HttpResponse';
+import { RenderHandler } from './RenderHandler';
+
+export class RenderEjsHandler<T> extends RenderHandler<T> {
+  private readonly ejsTemplatePath: string;
+  private readonly viewsFolder: string;
+
+  public constructor(viewsFolder: string, ejsTemplatePath: string) {
+    super();
+    this.viewsFolder = viewsFolder;
+    this.ejsTemplatePath = ejsTemplatePath;
+  }
+
+  public async handle(input: {
+    response: HttpResponse;
+    props: T;
+  }): Promise<void> {
+    const { props, response } = input;
+    const renderedHtml = await renderFile(
+      path.join(
+        this.viewsFolder,
+        this.ejsTemplatePath,
+      ),
+      props || {},
+    );
+    // Content-Type must not be cammel case
```

Comment:
typo

Reply:
but why not though? we have it in other places

Reply:
Really? This isn't for internal use. The 'content-type' here is the actual header. Is the 'content-type' header allowed to be camel case?